### ISSUE TITLE
How to model a different departure than arrival platform?

### ISSUE THREAD

**Issue by skinkie** [NeTEx-CEN/NeTEx#738]

**Comment by ue71603** [comment]
With Calls no problem. What do you really want to know?

**Comment by ue71603** [comment]
show be doable in Arrival see 10.6: https://www.xn--v-info-vxa.ch/sites/default/files/2024-05/NeTEx_Core-Realisation_Guide_TP_Suisse-v1.00.pdf

**Comment by skinkie** [comment]
> With Calls no problem. What do you really want to know?

I would argue that with a Call it is a problem too, because a Call has a ScheduledStopPointRef, not the individual Arrival or Departure. Hence if you want to do it by calls you also have to split it in two calls.

**Comment by ue71603** [comment]
You are right. We had the discussion with Laurent. And that's why he thinks HRDF is superior (one of the reasons). What I think is that in NeTEx. This should always be modeled as two calls as there is manoeuver (splitting or a DeadRun).  
So I would really be interested, what @Aurige  or @nick-knowles says.

**Comment by skinkie** [comment]
I think there is a difference between JourneyPlanner, Passenger, Driver and Vehicle views on the same dataset. But now we are talking HRDF; some magic *Something allows for this special case?

**Comment by ue71603** [comment]
HRDF is very bad for this. The GLEISE are originally texts. We had to implement an extension to be able to store the sloids.

**Comment by skinkie** [comment]
Then I don't see the superiority of HRDF :-)

**Comment by ue71603** [comment]
@Aurige  @nick-knowles  Sorry, it doesn't make sense to rename TRAIN STOP ASSIGNMENT to PASSENGER BOARDING POSITION ASSIGNMENT. Especially as it can also link only a STOP PLACE with a TRAIN COMPONENT. No BOARDING POSITION at all involved. Most will be on QUAY level and almost nobody uses BOARDING POSITION anywhere.

Also if one is doing EPIP, then there are no CALL or CALL PART. So where is the assignment done for the arrival and departure?

**Comment by Aurige** [comment]
The **PassingTime** is for **ScheduledStopPoint**, the **Quay**/**StopPlace** assignment is expected to be provided by **PassengerStopAssignment**/**DynamicStopAssignment**. These StopAssignment have a _boardingUse_/_alightingUse_, so you can specialise them and have boarding and alighting on 2 different **Quays** (and you can of course define multiple StopAssignment).

**Comment by skinkie** [comment]
@Aurige is your suggestion to extend the ServiceJourneyPattern?

**Comment by Aurige** [comment]
No, why ? I think that we already have something that works to assign a different Quay to arrival and departure

**Comment by Aurige** [comment]
Anybody to provide the Example ? @nick-knowles , @Aurige @Ulf9  ?
